### PR TITLE
Mailing list dg tests

### PR DESCRIPTION
**Pull request by cookiedan42**
Manual testing improvements

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -954,7 +955,7 @@ testers are expected to do more *exploratory* testing.
        Expected: No contact is added due to invalid email format. Error details shown in the status message.
 
     1. Other incorrect add commands to try: `add n/James! <other valid params>`, `add <one invalid param amongst other valid params>`, `...`.<br>
-       Expected: Similar to previous.
+       Expected: No contact is added. Error details shown in the status message.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:958]: are shown?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -967,7 +968,7 @@ testers are expected to do more *exploratory* testing.
        Expected: No changes made. Error details shown in the status message.
 
     1. Other incorrect untag commands to try: `find`, `find n/<non existent name>`, `...`.<br>
-       Expected: Similar to previous.
+       Expected: No changes made. Error details shown in the status message.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:971]: same ^

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -980,7 +981,7 @@ testers are expected to do more *exploratory* testing.
        Expected: No changes made. Error details shown in the status message.
 
     1. Other incorrect untag commands to try: `find`, `find t/<non existent tag>`, `find t/<non existent tag> c/`, `...`.<br>
-       Expected: Similar to previous.
+       Expected: No changes made. Error details shown in the status message.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:984]: same ^

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -1007,7 +1008,7 @@ testers are expected to do more *exploratory* testing.
       Expected: No contact is deleted. Error details shown in the status message.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the size of the contacts that are currently being viewed).<br>
-      Expected: Similar to previous.
+      Expected: No contact is deleted. Error details shown in the status message.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1011]: same ^

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -1033,7 +1034,7 @@ testers are expected to do more *exploratory* testing.
        Expected: No contact is deleted since the end index is smaller than the start index. Error details are shown in the status message. The status bar remains the same.
 
     1. Other incorrect delete commands to try: `deletem`, `deletem -2 - -1 `, `deletem x - y`, `...` (where x is not less than y, either of x or y is not a positive integer within the list size currently being viewed).<br>
-       Expected: Similar to previous.
+       Expected: No changes made. Error details shown in the status message.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1037]: same ^

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -1048,7 +1049,7 @@ testers are expected to do more *exploratory* testing.
        Expected: No changes made. Error details shown in the status message.
 
     1. Other incorrect pin commands to try: `pin`, `pin -1`, `pin x`, `...` (where x is larger than the size of the contacts that are currently being viewed).<br>
-       Expected: Similar to previous. 
+       Expected: No changes made. Error details shown in the status message.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1052]: same ^

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -1062,22 +1063,25 @@ testers are expected to do more *exploratory* testing.
        Expected: No changes made. Error details shown in the status message.
 
     1. Other incorrect unpin commands to try: `unpin`, `unpin -1`, `unpin x`, `...` (where x is larger than the size of the contacts that are currently being viewed).<br>
-       Expected: Similar to previous.
+       Expected: No changes made. Error details shown in the status message.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1066]: same ^

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -1062,22 +1063,25 @@ testers are expected to do more *exploratory* testing.
...
     1. Test case: `tag 2 t/<existing tag in contact at index 2>`<br>
-       Expected: No changes made. Error details shown in the status message.
+       Expected: No changes to the contact. Warning details shown in the status message.
+       
+    1. Test case: `tag 2 t/<tag not in contact at index 2 > t/<existing tag in contact at index 2>`<br>
+        Expected: New tag is added to the contact. Warning details shown in the status message.
 
     1. Other incorrect tag commands to try: `tag`, `tag -1 t/<valid tag name>`, `tag <tag name longer than 60 characters>`, `...`.<br>
-       Expected: Similar to previous.
+       Expected: No changes to contacts. Error details in status message.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1084]: same ^

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -1091,21 +1095,21 @@ testers are expected to do more *exploratory* testing.
        Expected: No changes made. Error details shown in the status message.
        
     1. Other incorrect untag commands to try: `untag`, `untag -1 t/<valid tag name>`, `...`.<br>
-       Expected: Similar to previous.
+       Expected: No changes made. Error details shown in the status message.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1098]: same ^

**Review by leezhixuan — APPROVED** [review]
some minor stuff, otherwise LGTM

File: docs/DeveloperGuide.md [hunk 10]
```diff
@@ -943,7 +943,8 @@ testers are expected to do more *exploratory* testing.
        Expected: No contact is added due to duplicated phone number. Error details shown in the status message.
 
     1. Other incorrect add commands to try: `add <other valid params> b/<future date>`, `add <other valid params> p/hp:1200123`, `...`.
-
+       Expected: No contact is added. Error details shown in the status message.
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:946]: Should have one line between `Expected` and the previous line or include a \<br> at the end of the previous line, if not `Expected` will become part of the previous line.
Screenshot 2021-11-08 at 12 51 14 AM

File: docs/DeveloperGuide.md [hunk 11]
```diff
@@ -1118,6 +1122,4 @@ testers are expected to do more *exploratory* testing.
         Expected: CONNECTIONS launches successfully with 0 contacts. 
        
     1. Other incorrect formatting of data or invalid fields: birthday to `00000000`, phone to `mynumber`, `...`
-        Expected: Similar to previous
-
-1. _{ more test cases …​ }_
+        Expected: CONNECTIONS launches successfully with 0 contacts.
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:1125]: Same here, should leave one line in between or use \<br>, if not looks like this
Screenshot 2021-11-08 at 12 53 49 AM

**Review by Jiwei99 — APPROVED** [review]
Some changes required for aesthetics, otherwise LGTM

**Pull request merged**